src/Entity/ShippingExportInterface.php

```diff
@@ -17,8 +17,9 @@
 
 interface ShippingExportInterface extends ResourceInterface
 {
-    const STATE_NEW = 'new';
-    const STATE_EXPORTED = 'exported';
+    public const STATE_NEW = 'new';
+
```

Comment:
If extra line is needed by standards, fine. But not sure if it's really needed.

Reply:
In my opinion extra line between const is not required but in Sylius Standard exist `\PhpCsFixer\Fixer\ClassNotation\ClassAttributesSeparationFixer` which have a default configuration for consts.
```php
->setDefault([
                    'const' => self::SPACING_ONE,
                    'method' => self::SPACING_ONE,
                    'property' => self::SPACING_ONE,
                ])
```
We can override it or create our own standard based on Sylius.

Reply:
keep it as it is, maybe topic to discuss for future